### DISCUSSION TITLE
Stopped working since Nov 1st, 2022

### DISCUSSION THREAD

**Discussion by mohataza333** [alandtse/alexa_media_player#1773] [General]
Did Amazon update their APIs?, this stopped working since Nov 1st, 2022
It's no longer discovering echo show/dot devices

**Comment by eabraga69** [comment]
I'm having the same problem. First it stopped working, I uninstalled it and installed it again but now, after entering the account data, password and confirmation, it only shows an "Error" message, without details, and it does not connect. Help me please.
